cumulus/tasks/queue-pdrs/tests/index.js

```diff
@@ -14,15 +14,17 @@ const workflowTemplate = require('./fixtures/workflow-template.json');
 const aws = require('@cumulus/common/aws');
 
 test.beforeEach(async (t) => {
-  t.context.bucket = testUtils.randomString();
-  await sqs().createQueue({ QueueName: 'testQueue' }).promise();
+  t.context.queueUrl = await createQueue();
+
+  t.context.bucket = randomString();
   return s3().createBucket({ Bucket: t.context.bucket }).promise();
 });
 
-test.afterEach.always(async (t) => {
-  await recursivelyDeleteS3Bucket(t.context.bucket);
-  await sqs().deleteQueue({ QueueUrl: `http://${process.env.LOCALSTACK_HOST}:4576/queue/testQueue` }).promise();
-});
+test.afterEach.always((t) =>
+  Promise.all([
```

Comment:
I think we should use async/await here too, unless, we don't really need to wait for this finish.

Reply:
Similar to what I added in my previous comment, ava will wait for the Promise returned by Promise.all to resolve.